openvdb/points/PointCount.h

```diff
@@ -117,6 +117,36 @@ Index64 inactiveGroupPointCount(const PointDataTreeT& tree, const Name& name,
     const bool inCoreOnly = false);
 
 
+/// @brief Total points in a tree which are included by a filter
+/// @param tree PointDataTree.
+/// @param filter Filter of the specified template type
+/// @param inCoreOnly if true, points in out-of-core leaf nodes are not counted
+template <typename PointDataTreeT, typename FilterT>
+Index64 filterPointCount(const PointDataTreeT& tree,
+    const FilterT& filter,
+    const bool inCoreOnly = false);
+
+
+/// @brief Total active points in a tree which are included by a filter
+/// @param tree PointDataTree.
+/// @param filter Filter of the specified template type
+/// @param inCoreOnly if true, points in out-of-core leaf nodes are not counted
+template <typename PointDataTreeT, typename FilterT>
+Index64 filterActivePointCount(const PointDataTreeT& tree,
```

Comment:
Probably out of the scope of this feature, but just to mention an inconsistency - in the PointDataGrid class there's an "onPointCount()" instead of "activePointCount()". Perhaps it would make sense to attempt to unify these.